packages/shared/src/components/modals/KeywordSynonymModal.tsx

```diff
@@ -48,49 +46,39 @@ export default function KeywordSynonymModal({
   };
 
   return (
-    <ResponsiveModal
-      className={classNames(className, styles.keywordSynonymModal)}
-      {...props}
-    >
-      <Button
-        onClick={props.onRequestClose}
-        buttonSize="small"
-        title="Close"
-        className="self-end btn-tertiary"
-      >
-        <XIcon />
-      </Button>
-      <h1 className="typo-callout">{`Find synonym for "${selectedKeyword}"`}</h1>
-      <TextField
-        inputId="search"
-        name="search"
-        label="Search for a keyword"
-        value={selectedKeyword}
-        autoFocus
-        valueChanged={onQueryChanged}
-        autoComplete="off"
-        className={{ container: 'self-stretch mb-3' }}
-      />
-      {!emptyResults && (
-        <ul className="flex flex-col gap-2 p-0 m-0 list-none">
-          {searchResults?.searchKeywords.hits.slice(0, 5).map((keyword) => (
-            <li className="p-0 m-0" key={keyword.value}>
-              <Button
-                onClick={() => setSynonym(keyword.value)}
-                className="btn-tertiary"
-              >
-                {keyword.value}
-              </Button>
-            </li>
-          ))}
-        </ul>
-      )}
-      <Button
-        className="self-start mt-2 btn-primary"
-        onClick={() => setSynonym(query)}
-      >
-        Create
-      </Button>
-    </ResponsiveModal>
+    <Modal kind={Modal.Kind.FlexibleTop} size={Modal.Size.Medium} {...props}>
+      <Modal.Header title={`Find synonym for "${selectedKeyword}"`} />
+      <Modal.Body>
+        <TextField
+          inputId="search"
+          name="search"
+          label="Search for a keyword"
+          value={selectedKeyword}
+          autoFocus
+          valueChanged={onQueryChanged}
+          autoComplete="off"
+          className={{ container: 'self-stretch mb-3' }}
```

Comment:
I think we should move the `margin` class to the below component `ul` to avoid unnecessary empty space below the textbox when it is empty

Reply:
Ah good one, it's only internal but moved it around :)